### PR TITLE
Tic Tac Toe JS

### PR DESCRIPTION
**Pull request by tirthajyoti-ghosh**
# Tic Tac Toe JS

### The classical game of tic tac toe is built collaboratively using HTML, a bit of CSS, and a whole lot of JS.
- Used modules and factories to separate code.
- Avoided using global code.
- Used factory function to initialize players with their names.
- Used modules to separate logic from DOM manipulation.
- Modules used: `board`, `game`, and `gameController`.
- Players have to input their names to start the game.

### CODE REVIEW

File: script/script.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-loop-func */
+/* eslint-disable no-unused-vars */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-undef */
```
Comment (mcrd25) [inline script/script.js]: Instead of disabling so many rules, try and address them. Maybe I can help. :slightly_smiling_face:

Comment (mcrd25) [reply script/script.js]: It improves your coding style as well and helps you keep the right standards

File: script/script.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    }
+  };
+
+  return {
+    run,
+  };
+})();
+
+
+function initPlayers(form) {
```
Comment (mcrd25) [inline script/script.js]: [OPTIONAL]
I noticed that you used ES6 for most of this app, maybe consider also making this follow ES6 syntax. :slightly_smiling_face:

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required :mag: 

Hi Team! :wave: 

You guys did an EXCELLENT job with the comments. :clap: Really. The UI is also very friendly!! 

I would have given you the pass but you've disabled quite a few rules to allow linter to pass. Try addressing them and if you have any issues feel free to email me or comment.

But really, great job! :smile: 

**Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: 

Thank you for applying the changes.
It's OK now. You can merge it :+1: 

Good continuation :handshake: 

Mael FOSSO

**Pull request merged**